HealthcareIS.sln

```diff
@@ -29,10 +33,14 @@ Global
 		{7CB35A4E-A104-4098-A2FE-23E4A42856B0}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{7CB35A4E-A104-4098-A2FE-23E4A42856B0}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{7CB35A4E-A104-4098-A2FE-23E4A42856B0}.Release|Any CPU.Build.0 = Release|Any CPU
-		{3EF55118-B41F-4E4F-BB2C-07BD77A46FCE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{3EF55118-B41F-4E4F-BB2C-07BD77A46FCE}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{3EF55118-B41F-4E4F-BB2C-07BD77A46FCE}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{3EF55118-B41F-4E4F-BB2C-07BD77A46FCE}.Release|Any CPU.Build.0 = Release|Any CPU
+		{9564E34A-6EF5-4E4E-A250-2A348C5A031D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{9564E34A-6EF5-4E4E-A250-2A348C5A031D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{9564E34A-6EF5-4E4E-A250-2A348C5A031D}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{9564E34A-6EF5-4E4E-A250-2A348C5A031D}.Release|Any CPU.Build.0 = Release|Any CPU
+		{51B18CC7-B280-4A8F-A019-CAE492B7ACDF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{51B18CC7-B280-4A8F-A019-CAE492B7ACDF}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{51B18CC7-B280-4A8F-A019-CAE492B7ACDF}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{51B18CC7-B280-4A8F-A019-CAE492B7ACDF}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
Why is this in PR?

Reply:
I don't even know what this represents. This file probably should be in .gitignore